### ISSUE TITLE
[BUG]: Error when using local lang. model support for OpenAPI spec generation

### ISSUE THREAD

**Issue by andrewconnell** [microsoft/dev-proxy#810]
### Description

Following docs:
- https://learn.microsoft.com/en-us/microsoft-cloud/dev/dev-proxy/how-to/generate-openapi-spec
- https://github.com/ollama/ollama/blob/main/README.md#ollama

When recording & saving a session, I received the error: **OllamaLanguageModelClient: Language model availability is not checked. Call IsEnabled first.**

```console
◉ Recording...

 req   ╭ GET https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?api_key=KzzpfOrja1LWM5QEoExF1d3CQNfGRjXv6WL8I7iw&sol=1000&camera=MAST
 api   ╰ Passed through
○ Stopped recording
 info    Creating OpenAPI spec from recorded requests...
 fail    OllamaLanguageModelClient: Language model availability is not checked. Call IsEnabled first.
 fail    OllamaLanguageModelClient: Language model availability is not checked. Call IsEnabled first.
 info    Created OpenAPI spec file api.nasa.gov-20240627153754.json
 info    DONE
```

### Expected behaviour

It creates the OpenAPI spec file using the local lang. model support. shown in this post: https://devblogs.microsoft.com/microsoft365dev/dev-proxy-v0-19-with-simulating-llm-apis-and-new-azure-api-center-integrations/?ocid=microsoft365dev_eml_tnp_autoid134_title

![](https://devblogs.microsoft.com/microsoft365dev/wp-content/uploads/sites/73/2024/06/word-image-21968-3.png)

### Actual behaviour

Error when saving the OpenAPI spec from the recorded requests:

```console
◉ Recording...

 req   ╭ GET https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?api_key=KzzpfOrja1LWM5QEoExF1d3CQNfGRjXv6WL8I7iw&sol=1000&camera=MAST
 api   ╰ Passed through
○ Stopped recording
 info    Creating OpenAPI spec from recorded requests...
 fail    OllamaLanguageModelClient: Language model availability is not checked. Call IsEnabled first.
 fail    OllamaLanguageModelClient: Language model availability is not checked. Call IsEnabled first.
 info    Created OpenAPI spec file api.nasa.gov-20240627153754.json
 info    DONE
```

### Steps to reproduce

1. install Dev Proxy & do the initial run to trust cert

    ```console
    brew tap microsoft/dev-proxy
    brew install dev-proxy
    ```

1. install Ollama & start service

    ```console
    brew install ollama
    brew services start ollama
    ```

1. verify Ollama is running and listening on default port by trying to start it again

    ```console
    ollama serve
    Error: listen tcp 127.0.0.1:11434: bind: address already in use
    ```

    > NOTE - after i got the **Actual Behavior** error above, i repeated the process but at this point, I had Ollama download & run the Phi3 model (`ollama run phi3`), but this had no effect - still got the same error.

1. update Dev Proxy config file to add the **OpenApiSpecGeneratorPlugin** plugin, update the `urlsToWatch`, & enable the local language model
1. start Dev Proxy, start recording

    ```console
    devproxy --failure-rate 0
    r
    ```

1. navigate to the following URL:  https://api.nasa.gov/mars-photos/api/v1/rovers/spirit/photos?api_key=DEMO_KEY&sol=1&page=1
1. stop recording

    ```console
    s
    ```

1. observe the error in the console... but the OpenAPI spec file is successfully created

### Dev Proxy Version

0.19.0

### Operating system (environment)

macOS

### Shell

zsh

### Configuration file

```json
{
  "$schema": "https://raw.githubusercontent.com/microsoft/dev-proxy/main/schemas/v0.19.0/rc.schema.json",
  "plugins": [
    {
      "name": "RetryAfterPlugin",
      "enabled": true,
      "pluginPath": "~appFolder/plugins/dev-proxy-plugins.dll"
    },
    {
      "name": "GenericRandomErrorPlugin",
      "enabled": true,
      "pluginPath": "~appFolder/plugins/dev-proxy-plugins.dll",
      "configSection": "genericRandomErrorPlugin"
    },
    {
      "name": "OpenApiSpecGeneratorPlugin",
      "enabled": true,
      "pluginPath": "~appFolder/plugins/dev-proxy-plugins.dll"
    }
  ],
  "urlsToWatch": [
    "https://jsonplaceholder.typicode.com/*",
    "https://api.nasa.gov/*"
  ],
  "genericRandomErrorPlugin": {
    "errorsFile": "devproxy-errors.json"
  },
  "rate": 50,
  "logLevel": "information",
  "newVersionNotification": "stable",
  "languageModel": { "enabled": true }
}
```

**Comment by garrytrinder** [comment]
Thanks for reporting @andrewconnell sorry for the trouble, we will take a look and get back to you soon!

**Comment by waldekmastykarz** [comment]
I can repro and most likely it's a bug on our end. Sorry for the trouble. Will fix asap

**Comment by waldekmastykarz** [comment]
Bug fixed in v0.19.1 available on Homebrew
